src/StreamWrapper/IncludeInterceptor.php

```diff
@@ -95,6 +95,8 @@ public function stream_open($path, $mode, $options)
 
     public function dir_closedir()
     {
+        \assert(\is_resource($this->fp));
```

Comment:
Another way to handle this would be declaring `$fp` as `@var resource` instead of `@var resource|bool`, but it could actually be `false` so best is to use actual asserts.